src/ucp/core/ucp_worker.c

```diff
@@ -1757,6 +1770,37 @@ static ucs_status_t ucp_worker_init_mpools(ucp_worker_h worker)
         goto err_reg_mp_cleanup;
     }
 
+    /* Create memory pool for pipelined rndv fragments on device*/
```

Comment:
need to remove rndv_frag_mp mpool above. it will be part of mem_type_mp with mem_type UCS_MEMORY_TYPE HOST.